### ISSUE TITLE
Make it harder to start the swipe

### ISSUE THREAD

**Issue by athenawisdoms** [jemise111/react-native-swipe-list-view#617]
**Describe the bug**
When the user is scrolling down the list, its very easy to trigger the start of the swipe, which causes the row/element to blink and momentarily show the component rendered by `renderHiddenRow`.

Is there a way to make it harder to trigger the start of the swipe, like setting how the user must swipe X pixels before starting the swipe?

Tried setting `swipeToOpenPercent={50}` as a `SwipeListView` prop but even a small sliding gesture still causes the swipe to trigger. Am not using `SwipeRow` in `renderItem`.

Thanks in advance :)

**Comment by pepperav** [comment]
Yes I agree with @athenawisdoms , same issues. The top/bottom scroll immediately cancel a swipe already started or completed. 

I solved setting `closeOnScroll={ false }`. Did you try?

Thank you and @jemise111  great work! 🙌
